packages/react-dom/src/client/ReactDOMHostConfig.js

```diff
@@ -19,11 +25,14 @@ import type {
 } from '../shared/ReactDOMTypes';
 import type {ReactScopeMethods} from 'shared/ReactTypes';
 
+import {getRole} from 'dom-accessibility-api';
```

Comment:
I don't have high hopes that this would be guaranteed to be tree shaken for non-testing builds. This also has a bunch of compiled Babel junk like `sliced` helpers. The entry point also contains another import which isn't used here. I think ideally we would have an inlined implementation.

Reply:
Yeah I understand. I thought about inlining it to begin with but decided I'd rather just make progress initially on the API and the tests, and then swap this out later on :smile:

Reply:
Yeah this is transpiled pretty far down to support IE 11. The `getRole` helper was originally intended to be a temporary drop-in replacement for aria-query which had some bugs that are now resolved. Ideally aria-query should be favored in the long run since it is used throughout the React ecosystem. 

Having another fork for implicit roles should be avoided IMO. Inconsistencies in those can be pretty frustrating.

Regarding ESM: This should be usable as an ES module. I have extra tests that verify this for node 13.

Reply:
I looked into using `aria-query` initially (since it seems like that's the main thing RTL is using) but the documentation was pretty non-existent, there weren't really any tests to refer to, and I wasn't sure how to properly use it.